website/docs/patterns/table-multi-select/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,33 @@
+!!! Info
+
+It is the responsibility of consumers to implement this pattern, and therefore also their responsibility to ensure that all accessibility success criteria are met or exceeded.
```

Comment:
```suggestion
It is the responsibility of consumers to implement this pattern and to ensure that all accessibility success criteria are met or exceeded.
```

Reply:
I recommend applying the above suggested edit to reduce repetition and clarify the language.

Reply:
Hmmm, I missed this one, thanks for the note!